share/shwap/row_namespace_data.go

```diff
@@ -37,22 +66,22 @@ func RowNamespaceDataFromShares(
 			break
 		}
 	}
+
+	// if count is 0, then the namespace is not present in the shares. Return non-inclusion proof.
 	if count == 0 {
-		// FIXME: This should return Non-inclusion proofs instead. Need support in app wrapper to generate
-		// absence proofs.
-		return RowNamespaceData{}, fmt.Errorf("no shares found in the namespace for row %d", rowIndex)
```

Comment:
So does it return non-inclusion proof now? If so, we may want to close the issue that we made for those in shwap context. Also, we should test that on Accessor if not yet

Reply:
Yes, I've added fucntionality here as it was required and I didn't want to make too many PRs. Issue is linked already, check description

Reply:
Do we test it ?